src/components/DonateButton.jsx

```diff
@@ -263,12 +263,12 @@ class DonateButton extends Component {
                   onChange={(name, value) => this.setState({ mewAmount: value })}
                   placeholder="10"
                   validations={{
-                    lessThan: wallet.getBalance() - 0.5,
-                    greaterThan: 0.00000000009,
+                    lessThan: wallet.getBalance(),
+                    greaterThan: 0.09,
```

Comment:
Would `0.091` be a valid amount? If so `Minimum value must be at least Ξ0.1` is not correct. Could we make this `greaterThanEqual: 0.10`?